configure.ac

```diff
@@ -84,7 +84,7 @@ GLIB_GSETTINGS
 
 dnl Specify required versions of dependencies
 CAIRO_REQUIRED=1.14.0
-GLIB_REQUIRED=2.54.0
+GLIB_REQUIRED=2.62.0
```

Comment:
Do we really need this version bump?
Than i can't use dev version of atril in f30 any more.
```
[rave@mother Schreibtisch]$ rpm -qa glib2
glib2-2.60.7-2.fc30.x86_64
```

Reply:
I guess it's because of `g_file_info_get_modification_date_time`...
https://developer.gnome.org/gio/unstable/GFileInfo.html#g-file-info-get-modification-date-time